Pipfile

```diff
@@ -27,7 +27,6 @@ python-coveralls = "*"
 mock = "*"
 pytest-mock = "*"
 vulture = "*"
-v = {version = "*",editable = true}
```

Comment:
Do you know what this line actually was for? It made pipenv crash when trying to write the lock file.

Reply:
No idea. @cleborys